src/pages/lesson-planning.tsx

```diff
@@ -259,11 +275,7 @@ const PlanALesson: NextPage<PlanALessonProps> = ({ pageData }) => {
                 <ButtonAsLink
                   icon="search"
                   $iconPosition="trailing"
-                  label={pageData.lessonElementsCTA.label}
-                  page="teacher-hub"
-                  htmlAnchorProps={{
-                    target: "_blank",
-                  }}
+                  {...generateCtaProps(pageData.lessonElementsCTA)}
```

Comment:
that's possible but the request was for it to be editable via Sanity so we wanted to get away from hardcoding it - is there a reason it would be better to hardcode it?

Reply:
Sorry i meant pass 'home' from sanity and pass it into the page prop ?

Reply:
Check the code base and yes  href={getLinkHref(cta)}  seems like the correct way to do this.